pysat/instruments/champ_star.py

```diff
@@ -29,6 +29,7 @@
 import numpy as np
 import pandas as pds
 import warnings
+from pysat.utils import NetworkLock as Lock
```

Comment:
Why import from pysat.utils here?

Reply:
Asher created a class called NetworkLock in pysat.utils._core that applies the fout.flush() and sync stuff needed for networked filesystems.

Reply:
It would be clearer to not nickname the function, then.